app/lib/frontend/templates.dart

```diff
@@ -124,6 +124,66 @@ class TemplateService {
         title: pageTitle, faviconUrl: faviconUrl);
   }
 
+  /// Renders the `views/v2/pkg/index.mustache` template.
+  String renderPkgIndexPageV2(List<PackageView> packages, PageLinks links,
+      String currentPlatform) {
+    final packagesJson = [];
+    for (int i = 0; i < packages.length; i++) {
+      final view = packages[i];
+      packagesJson.add({
+        'url': '/packages/${view.name}',
+        'name': view.name,
+        'version': HTML_ESCAPE.convert(view.version),
+        'show_dev_version': view.devVersion != null,
+        'dev_version': HTML_ESCAPE.convert(view.devVersion ?? ''),
+        'dev_version_href': Uri.encodeComponent(view.devVersion ?? ''),
+        'last_uploaded': view.shortUpdated,
+        'desc': view.ellipsizedDescription,
+        'tags_html': _renderTags(view.platforms), // experimental_design: new
+        // TODO: add authors (?) - old version had it, new design may not have the space for it
+      });
+    }
+
+    String title = 'Packages';
+    String descriptionHtml = '';
+    if (currentPlatform == KnownPlatforms.flutter) {
+      title = 'Flutter Packages';
+      descriptionHtml = flutterPackagesDescriptionHtml;
+    }
+
+    Map platformTabData(String text, String platform) {
+      final String param = platform == null
+          ? null
+          : new PlatformPredicate(required: [platform]).toQueryParamValue();
+      final String query = param == null ? '' : '?platform=$param';
+      final String url = '/packages$query';
+      return {'text': text, 'href': url, 'active': platform == currentPlatform};
+    }
+
+    final platformTabs = [
+      platformTabData('Flutter', KnownPlatforms.flutter),
+      platformTabData('Web', KnownPlatforms.web),
+      platformTabData('Server', KnownPlatforms.server),
+      platformTabData('All', null),
+    ];
+
+    final values = {
+      'platform_tabs': platformTabs, // experimental_design: new
```

Comment:
I don't even understand what `// experimental_design: new` means. Maybe just get rid of these comments. If the map entries are used in old/new templates, maybe rephrase these to `// used in v2 only`.

Reply:
yeah, I'll do a followup to change everything. it was meant to be a unique string that is easy to search for when we'll do the merging and cleanup, a shorter version works just as well. (initially I was aiming for naming everything as "experimental", but that is no longer the case)